OpenRA.Game/Traits/Player/FrozenActorLayer.cs

```diff
@@ -129,7 +138,7 @@ public IEnumerable<IRenderable> Render(WorldRenderer wr)
 			return renderables;
 		}
 
-		public bool HasRenderables { get { return renderables.Any(); } }
+		public bool HasRenderables { get { return !Shrouded && renderables.Any(); } }
```

Comment:
Likewise here - if we don't do this the actor will be treated the same as if it were not shrouded or fogged.